VariablesHomework/VariablesHomework.cs

```diff
@@ -60,86 +60,86 @@ public void Operators()
         {
             int a = 0;
             int b = a + 13; 
-            Assert.IsTrue(b == /*Answer here*/); // Make this pass
+            Assert.IsTrue(b == 13); // Make this pass
 
             float f = 12.3f;
             float c = f - 7.43f;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass
-            Assert.IsTrue(f == /*Answer here*/); // Make this pass
+            Assert.IsTrue(c == 4.8700004f); // Make this pass
+            Assert.IsTrue(f == 12.3f); // Make this pass
 
             bool d = false;
             bool e = !d;
-            Assert.IsTrue(e == /*Answer here*/); // Make this pass
+            Assert.IsTrue(e == true); // Make this pass
 
             int anotherInt = 12;
             int g = -anotherInt; 
-            Assert.IsTrue(g == /*Answer here*/); // Make this pass
+            Assert.IsTrue(g == -12); // Make this pass
 
             int yetAnotherInt = 23456;
             int t = yetAnotherInt * 3;
-            Assert.IsTrue(t == /*Answer here*/); // Make this pass
+            Assert.IsTrue(t == 70368); // Make this pass
 
             int z = 23456;
             z *= 3;
-            Assert.IsTrue(z == /*Answer here*/); // Make this pass
+            Assert.IsTrue(z == 70368); // Make this pass
 
             int h = 23;
             h++;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 23); // Make this pass
```

Comment:
The test is failing on this line. A couple of the other Asserts for `h == /*Answer here*/` also wouldn't pass.